Test/dafny0/Bitvectors.dfy.expect

```diff
@@ -100,57 +100,57 @@ module _System {
 */
 
 /* CALL GRAPH for module _module:
- * SCC at height 27:
+ * SCC at height 0:
  *   M
- * SCC at height 26:
+ * SCC at height 3:
  *   Main
- * SCC at height 25:
+ * SCC at height 2:
```

Comment:
Changes like this shows really good improvements of removing dependencies. Excellent!